spec/_attachments/ic.did

```diff
@@ -184,6 +192,14 @@ service ic : {
   bitcoin_send_transaction: (send_transaction_request) -> ();
   bitcoin_get_current_fee_percentiles: (get_current_fee_percentiles_request) -> (vec millisatoshi_per_byte);
 
+  // metrics interface
+  metrics : (record {
+    subnet_id : principal;
+  }) -> (record {
+    timestamp_nanos : nat64;
```

Comment:
based on the other discussion:
```suggestion
    start_timestamp_nanos : nat64;
```

Reply:
I'd leave it as is, see https://github.com/dfinity/interface-spec/pull/215/files#r1318372688

Reply:
Indeed.